src/gui/layout/qgslayoutview.h

```diff
@@ -337,6 +337,20 @@ class GUI_EXPORT QgsLayoutView: public QGraphicsView
      */
     void zoomActual();
 
+    /**
+     * Zooms to the previous zoom level
+     * \since QGIS 3.38
```

Comment:
Should be 3.40 instead?
Or 3.42, since we're already in FF.

Reply:
3.42 please!

Reply:
My bad. 3.42 it is!